config/field_test.yml

```
@@ -28,10 +28,34 @@
 ################################################################################
 experiments:
   # NOTE: Our feed strategy testing experiment must begin with "feed_strategy"
-  feed_strategy_starting_20220801:
+  feed_strategy_starting_20220823:
     # NOTE: Required as we want only want to consider for conversion events that
     #       occurred on or after the given start_date.
     started_at: 2022-08-01
```

wrong start time? :laughing: 
```suggestion
    started_at: 2022-08-03
```

---

I swear I actually edited this file correctly! Somehow I buggered it up!